cs3/ocm/core/v1beta1/ocm_core_api.proto

```diff
@@ -50,11 +52,20 @@ import "cs3/types/v1beta1/types.proto";
 // Any method MAY return UNKNOWN.
 // Any method MAY return UNAUTHENTICATED.
 service OcmCoreAPI {
-  // Creates a new ocm share.
+  // Creates a new OCM share. The share SHALL be sent to the remote system
+  // using the OCM API at:
+  // https://cs3org.github.io/OCM-API/docs.html?branch=v1.1.0&repo=OCM-API&user=cs3org#/paths/~1shares/post
   rpc CreateOCMCoreShare(CreateOCMCoreShareRequest) returns (CreateOCMCoreShareResponse);
+  // Updates an OCM share. This action SHALL be notified to the remote system
```

Comment:
Actually if we reach this, the remote is already been notified, as this is called in the "notified" side, i.e. the recipient provider of the share

Reply:
Correct, my last commit puts the comments in the right place